components/tags-modal.tsx

```
@@ -0,0 +1,100 @@
+          </p>
+        </ModalHeader>
+        <ModalBody>
+          <div className="flex flex-wrap gap-2 max-h-60 overflow-y-auto">
+            {selectedTags.map((tagId) => {
+              const tag = tags.find((t) => t.id === tagId);
+              return tag ? (
+                <span
+                  key={tagId}
+                  className="inline-flex items-center gap-1 px-3 py-2 text-sm bg-blue-100 dark:bg-blue-600/20 border border-blue-300 dark:border-blue-500 text-blue-700 dark:text-blue-400 rounded-lg transition-colors duration-300 hover:bg-blue-200 dark:hover:bg-blue-600/30 cursor-pointer"
+                  onClick={() => onRemoveTag(tagId)}
+                >
```

Biome (1.9.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)